packages/reporter/src/index.ts

```diff
@@ -0,0 +1,328 @@
+  const lightClientAddresses = settings.contractAddresses.lightClientAddresses as any
+
+  const multiClient = new Multiclient({
+    chains: [sourceChain, ...destinationChains],
+    privateKey: process.env.PRIVATE_KEY as `0x${string}`,
+    rpcUrls: settings.rpcUrls,
+  })
+
+  const ambReporterController = new StandardReporterController({
+    name: "AMBReporterController",
+    type: "classic",
+    sourceChain,
+    destinationChains: destinationChains.filter(({ name }) => name === gnosis.name || name === gnosisChiado.name),
+    logger,
+    multiClient,
+    reporterAddresses: {
+      [gnosis.name]: unidirectionalReportersAddresses[sourceChain.name]?.[gnosis.name]?.AMBReporter,
+      [gnosisChiado.name]: unidirectionalReportersAddresses[sourceChain.name]?.[gnosisChiado.name]?.AMBReporter,
+    },
+    adapterAddresses: {
+      [gnosis.name]: unidirectionalAdaptersAddresses[sourceChain.name]?.[gnosis.name]?.AMBAdapter,
+      [gnosisChiado.name]: unidirectionalAdaptersAddresses[sourceChain.name]?.[gnosisChiado.name]?.AMBAdapter,
+    },
+  })
+
+  const sygmaReporterController = new SygmaReporterController({
+    type: "classic",
+    sourceChain,
+    destinationChains,
+    logger,
+    multiClient,
+    reporterAddresses: {
+      [gnosis.name]: unidirectionalReportersAddresses[sourceChain.name]?.[gnosis.name]?.SygmaReporter,
+    },
+    adapterAddresses: {
+      [gnosis.name]: unidirectionalAdaptersAddresses[sourceChain.name]?.Gnosis?.SygmaAdapter,
+    },
+  })
+
+  const telepathyReporterController = new TelepathyReporterController({
+    type: "lightClient",
+    sourceChain,
+    destinationChains,
+    logger,
+    multiClient,
+    adapterAddresses: {
+      [gnosis.name]: unidirectionalAdaptersAddresses[sourceChain.name]?.Gnosis?.TelepathyAdapter,
+      [arbitrum.name]: unidirectionalAdaptersAddresses[sourceChain.name]?.["Arbitrum One"]?.TelepathyAdapter,
+      [optimism.name]: unidirectionalAdaptersAddresses[sourceChain.name]?.["OP Mainnet"]?.TelepathyAdapter,
+      [bsc.name]: unidirectionalAdaptersAddresses[sourceChain.name]?.["BNB Smart Chain"]?.TelepathyAdapter,
+      [polygon.name]: unidirectionalAdaptersAddresses[sourceChain.name]?.Polygon?.TelepathyAdapter,
+    },
+    baseProofUrl: settings.reporterControllers.TelepathyReporterController.baseProofUrl,
+    lightClientAddresses: {
+      [gnosis.name]: lightClientAddresses.Gnosis?.[sourceChain.name]?.TelepathyLightClient,
+      [arbitrum.name]: lightClientAddresses["Arbitrum One"]?.[sourceChain.name]?.TelepathyLightClient,
+      [optimism.name]: lightClientAddresses["OP Mainnet"]?.[sourceChain.name]?.TelepathyLightClient,
+      [bsc.name]: lightClientAddresses["BNB Smart Chain"]?.[sourceChain.name]?.TelepathyLightClient,
+      [polygon.name]: lightClientAddresses.Polygon?.[sourceChain.name]?.TelepathyLightClient,
+    },
+  })
+
+  const wormholeReporterController = new WormholeReporterController({
```

Comment:
same here

Reply:
same reason explained in https://github.com/gnosis/hashi/pull/66#discussion_r1764795977